config/settings/pen.yml

```
@@ -0,0 +1,51 @@
+# The url for this app, also used by `dfe_sign_in`
+base_url: https://pen.register-trainee-teachers.service.gov.uk
+
+dqt:
+  base_url: https://preprod.teacher-qualifications-api.education.gov.uk
+
+trs:
+  base_url: https://preprod.teacher-qualifications-api.education.gov.uk/
+
+dfe_sign_in:
+  # URL that the users are redirected to for signing in
+  issuer: https://oidc.signin.education.gov.uk
+  # URL of the users profile
+  profile: https://profile.signin.education.gov.uk
+
+recruits_api:
+  base_url: "https://sandbox.apply-for-teacher-training.service.gov.uk/register-api"
+
+features:
+  basic_auth: false
+  sign_in_method: dfe-sign-in
+  enable_feedback_link: false
+  import_courses_from_ttapi: true
+  import_applications_from_apply: true
+  publish_course_details: true
+  routes:
+    assessment_only: true
+    early_years_assessment_only: true
+    early_years_postgrad: true
+    early_years_salaried: true
+    early_years_undergrad: true
+    opt_in_undergrad: true
+    pg_teaching_apprenticeship: true
+    provider_led_postgrad: true
+    provider_led_undergrad: true
+    school_direct_salaried: true
+    school_direct_tuition_fee: true
+    iqts: true
+  integrate_with_dqt: true
+  integrate_with_trs: false
+  register_api: true
+  google:
+    send_data_to_big_query: false
+
```

We also want to enable the API token management feature, so may as well do it in this PR.
```suggestion
  token_management: true

```

---

added feature